packages/client/tailwindTheme.ts

```diff
@@ -154,6 +154,20 @@ export default {
         '800': '#2D2D39',
         '900': '#1C1C21'
       }
+    },
+    keyframes: {
+      overlayShow: {
+        from: {opacity: 0},
+        to: {opacity: 1}
+      },
+      contentShow: {
+        from: {opacity: 0, transform: 'translate(-50%, -48%) scale(0.96)'},
```

Comment:
+1 i like subtle animations but 2% translation on a dialog that is 150px tall is.... 3px 😄 . Too little!

A fun css rule is that you can pass in 2 translate functions & they'll be applied sequentially. that way you can keep your % based translate & then use one with pixels.

Try this:

```ts
contentShow: {
  from: {opacity: 0, transform: 'translate(-50%, -50%) translateY(32px) scale(0.96)'}
}
```

Since the animation finishes at the same position as the css class, no `to` is needed 🎉